PriceCalculatorKata/src/PriceCalculator/Product.cs

```diff
@@ -57,17 +24,25 @@ public double CalculatePriceAfterTax()
 
     public double CalculateTaxValue()
     {
-        return Price * (_tax.TaxValue / 100.0);
+        var tax = Tax.GetTax();
+        return Price * (tax.TaxValue / 100.0);
     }
 
     public double CalculateDiscountValue()
     {
-        return Price * (_relativeDiscount.DiscountValue / 100.0);
+        var universalDiscount = RelativeDiscount.GetDiscountInstance();
```

Comment:
I'm confused by the naming the relative discount is the same as universal discount ?